k8s/operator/manifests/operator.yaml

```diff
@@ -127,9 +127,6 @@ spec:
           containerPort: 9464
           protocol: TCP
         env:
-        # We are pointing to tempo or grafana tracing agent's otlp grpc receiver port
-        - name: OPERATOR_OTLP_ENDPOINT
-          value: "https://otel:4317"
```

Comment:
We need to remove this config option from any operators we have configured otherwise they will still try to send traces.